frontend/lib/resolution/VarScopeVisitor.h

```diff
@@ -140,6 +153,22 @@ class VarScopeVisitor {
       Only appropriate for expressions (not for Loops) */
   void handleMentions(const AstNode* ast, RV& rv);
 
+  /** Handle updating initedVars if an assignment represents a split-init.
+      Returns true if it was a split init. */
+  bool handleSplitInitAssign(const OpCall* ast,
```

Comment:
Most of your other `handle` methods are `virtual` and for children to override. This is not one of these methods. I think the `handle` name is partially confusing; when I see a call to it in a child, it makes me think that it's a child-specific implementation, rather than a more general parent method.

Reply:
Yes that's a reasonable point. `handleMentions` is in the same category. I can rename them.